services/web/server/requirements/_test.txt

```diff
@@ -22,15 +22,15 @@ atomicwrites==1.3.0       # via pytest
 attrs==19.1.0
 billiard==3.6.0.0
 celery==4.3.0
-certifi==2019.3.9         # via requests
+certifi==2019.9.11        # via requests
 cffi==1.12.3
 change-case==0.5.2
 chardet==3.0.4
 codecov==2.0.15
 coverage==4.5.3           # via codecov, coveralls, pytest-cov
 coveralls==1.7.0
 cryptography==2.6.1
-docker==4.0.1
+docker==4.1.0
```

Comment:
Did you need this upgrade for your code? If not please revert
I would separate PR with library upgrades ... otherwise it is messy.